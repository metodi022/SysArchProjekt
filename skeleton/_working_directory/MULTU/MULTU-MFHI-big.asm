addu  $1, $0, 99	# initialise memory
addu  $0, $0, 120	# initialise memory
multu $1, $2		# multiply $1 and $2
mfhi $3