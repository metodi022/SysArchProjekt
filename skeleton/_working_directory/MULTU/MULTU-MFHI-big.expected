@3 00101110
